prediction_market_agent/agents/arbitrage_agent/deploy.py

```diff
@@ -108,7 +108,7 @@ def get_correlated_markets(self, market: AgentMarket) -> list[CorrelatedMarketPa
             },
         )
 
-        omen_markets = self.subgraph_handler.get_omen_binary_markets(
+        omen_markets = self.subgraph_handler.get_omen_markets(
```

Comment:
renamed `get_X_binary_markets` -> `get_X_markets` (also true for other market types)